framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/Modeling/AbpEntityTypeBuilderExtensions.cs

```diff
@@ -61,7 +63,11 @@ public static void TryConfigureExtraProperties(this EntityTypeBuilder b)
                         d => JsonConvert.SerializeObject(d, Formatting.None),
                         s => JsonConvert.DeserializeObject<Dictionary<string, object>>(s)
                     )
-                    .HasColumnName(nameof(IHasExtraProperties.ExtraProperties));
+                    .HasColumnName(nameof(IHasExtraProperties.ExtraProperties))
+                    .Metadata.SetValueComparer(new ValueComparer<Dictionary<string, object>>(
+                        (d1, d2) => d1.SequenceEqual(d2),
+                        d => d.Aggregate(0, (k, v) => HashCode.Combine(k, v.GetHashCode())),
+                        d => d.ToDictionary(k => k.Key, v => v.Value)));
```

Comment:
Shall we extract `.HasConversion` and `.Metadata.SetValueComparer` to a new `HasJsonConversion` extension method in `AbpPropertyBuilderExtensions`?

Usage:

```c#
b.Property<Dictionary<string, object>>(nameof(IHasExtraProperties.ExtraProperties))
    .HasJsonConversion()
    .HasColumnName(nameof(IHasExtraProperties.ExtraProperties));
```

Reply:
In general, I will consider extracting it only if the code is repeated three times or more. I will try to extract it. Thank you @acjh